packages/react/src/isomorphicClerk.ts

```diff
@@ -287,15 +286,6 @@ export class IsomorphicClerk implements IsomorphicLoadedClerk {
     }
   };
 
-  buildHomeUrl = (): string | void => {
-    const callback = () => this.clerkjs?.buildHomeUrl() || '';
-    if (this.clerkjs && this.#loaded) {
-      return callback();
-    } else {
-      this.premountMethodCalls.set('buildHomeUrl', callback);
-    }
-  };
-
```

Comment:
🔧 We should also introduce the `buildAfterSignInUrl` and `buildAfterSignUpUrl` here.

Reply:
Done!